src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -509,6 +512,11 @@ namespace winrt::TerminalApp::implementation
             _mruTabs.RemoveAt(mruIndex);
         }
 
+        if (tab == _settingsTab)
+        {
+            _settingsTab = nullptr;
+        }

```

Comment:
I felt like this is a more robust way to express the same idea of "there's just one instance of settings and closing it destroys it".